### PR TITLE
Veremark

### PR DESCRIPTION
**Pull request by daallgeier**

### CODE REVIEW

File: _data/vendors.yml [hunk 1]
```diff
@@ -417,6 +417,15 @@
   pricing_source: https://www.twilio.com/enterprise
   updated_at: 2018-10-22
 
+- name: Veremark
+  url: https://www.veremark.com
+  base_pricing: pay as you go
+  sso_pricing: £100 per month
+  percent_increase: 100% [^veremark]
```
Comment (mbainter) [inline _data/vendors.yml:424]: Technically, this is 9900% increase in pricing, if you're going by 1GBP -> 100GBP.  (100% increase would be 2GBP)

**Review by mbainter — CHANGES_REQUESTED** [review]

File: _data/vendors.yml (hunk 1 above)
Comment (mbainter) [reply _data/vendors.yml:424]: It's a fair point, though I feel like the 100% understates how significant the tax is. perhaps the right answer here is to use the infinite symbol instead?

Comment (daallgeier) [reply _data/vendors.yml:424]: ∞ works too, i guess.

Comment (daallgeier) [reply _data/vendors.yml]: i added ∞ now lol

**Review by mbainter — APPROVED** [review]

**Pull request merged**